validator/node/node.go

```diff
@@ -232,11 +234,13 @@ func (s *ValidatorClient) initializeFromCLI(cliCtx *cli.Context) error {
 	if err := s.registerClientService(keyManagerV1, keyManagerV2); err != nil {
 		return err
 	}
-	if err := s.registerRPCService(cliCtx); err != nil {
-		return err
-	}
-	if err := s.registerRPCGatewayService(cliCtx); err != nil {
-		return err
+	if cliCtx.Bool(flags.EnableRPCFlag.Name) {
```

Comment:
```suggestion
	if cliCtx.Bool(flags.EnableRPCFlag.Name) || cliCtx.Bool(flags.EnableWebFlag) {
```

Reply:
RPC needs to be enabled for web to work.

Reply:
all services requried for webUI to work initialized in dedicated initializeForWeb() ?

Reply:
I see. Thanks!